src/components/BorrowerProfile/LoanDescription.vue

```diff
@@ -1,118 +1,149 @@
 <template>
 	<section>
-		<h2
-			class="tw-text-h2"
-		>
-			{{ firstName }}'s story
-		</h2>
-		<div
-			v-if="partnerName"
-		>
-			<div>
-				{{ moreInfoAboutLoan }}
-			</div>
-			<div
-				v-if="loanAlertText"
+		<div>
+			<h2
+				class="tw-text-h2"
```

Comment:
Since this is already an h2, I think you don't need to add the `tw-text-h2` class. The same should go for the other h2 and h3 elements in this pr, and the `tw-text-base` class on the p elements.